apps/calendar/style/multi_day.css

```diff
@@ -477,3 +480,81 @@
   /* reset text-indent to show the "+" icon */
   text-indent: 0;
 }
+
+/* RTL View */
+html[dir="rtl"] .day-view .md__all-day {
+  left: unset;
+  right: 0;
+}
+
+html[dir="rtl"] .day-view .icon-calendar-allday:before {
+  left: unset;
+  right: 2rem;
+}
+
+html[dir="rtl"] .md__allday-events {
+  border-left: unset;
+  border-right: 0.1rem solid #e2e2e2;
+}
+
+html[dir="rtl"] .day-view .md__display-hour {
+  text-align: right;
+}
+
+html[dir="rtl"] .md__days-wrapper,
+html[dir="rtl"] .md__alldays-wrapper {
+  left: 0;
+  right: unset;
+}
+
+html[dir="rtl"] .md__day,
+html[dir="rtl"] .md__allday {
+  float: right;
```

Comment:
it is not displaying the days on the week/day views.. during the swipe I can see some days/events but after release it doesn't snap to proper position.